### PR TITLE
train fixes

### PR DESCRIPTION
**Pull request by alexholdenmiller**

### CODE REVIEW

**Comment by alexholdenmiller** [comment]
seq2seq train:
`[ time:2s parleys:21 total_exs:84 time_left:51s ] {'total': 84, 'accuracy': 0.0119, 'f1': 0.06746, 'hits@k': {1: 0.0119, 5: 0.0119, 10: 0.0119, 50: 0.0119, 100: 0.0119}}`

drqa train:
`[ time:15s parleys:1 ] [train] updates = 1 | train loss = 9.68 | exs = 32`

**Comment by alexholdenmiller** [comment]
all tests and check_examples passing
